src/broker.js

```diff
@@ -10,7 +10,7 @@ createBroker = (data) => {
 		name : data.bc_firm_name,
 		otherNames : data.bc_other_names,
 		finraApprovedRegistrationCount : data.bc_approved_finra_registration_count,
-		branchLocations : data.bc_branches.map((branch)=>{return createBranch(branch)}),
+		branchLocations : (data.bc_branches && data.bc_branches.length > 0)?data.bc_branches.map((branch)=>{return createBranch(branch)}):[],
```

Comment:
@kamidipreetham 
This looks good. Instead of checking both for a value in `data.bc_branches` and `data.bc_branches.length>0`, could we simplify by only checking the list length? I think it may accomplish the same.
Like so:
`branchLocations : (data.bc_branches.length > 0)?data.bc_branches.map((branch)=>{return createBranch(branch)}):[],`

What are your thoughts?

Thanks!

Reply:
If data.bc_branches is null, the method length over it would fail. Hence, the double check.